examples/generator/arithmetic.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+from pyqir.generator import BasicQisBuilder, SimpleModule, const, types
+
+mod = SimpleModule("arithmetic", num_qubits=0, num_results=0)
+qis = BasicQisBuilder(mod.builder)
+
+# Declare functions that can produce and consume integers at runtime. See
+# external_functions.py.
+i32 = types.Int(32)
+get_int = mod.add_external_function("get_int", types.Function([], i32))
+take_int = mod.add_external_function(
+    "take_int", types.Function([i32], types.VOID))
+
+# Add 3 to a number and multiply the result by 2.
+a = mod.builder.call(get_int, [])
+b = mod.builder.add(const(i32, 3), a)
```

Comment:
@samarsha I think we should pull `const(i32, 3)` here out and explain why `const` is being called.